src/graph/test/SchemaTest.cpp

```diff
@@ -184,21 +203,28 @@ TEST_F(SchemaTest, metaCommunication) {
         };
         ASSERT_TRUE(verifyResult(resp, expected));
     }
+    // Test create edge without prop
     {
-        // Test tag not exist
         cpp2::ExecutionResponse resp;
-        std::string query = "DESCRIBE TAG not_exist";
+        std::string query = "CREATE EDGE edge1()";
```

Comment:
add case:
alter edge1 without property to add a property.

Reply:
good